### PR TITLE
Fix attach not working in docker studio

### PR DESCRIPTION
**Pull request by elliott-davis**
Signed-off-by: Elliott Davis <elliott@excellent.io>

Closes: #5481

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/hab/plan.sh [hunk 1]
```diff
@@ -23,6 +23,7 @@ pkg_bin_dirs=(bin)
 bin=$_pkg_distname
 
 _common_prepare() {
+  attach
```
Comment (baumanj) [inline components/hab/plan.sh]: I'm not very familiar with this. Can you explain why it wasn't necessary before? Or was this always missing?

**Comment by elliott-davis** [comment]
Oh snap. I put that in for testing and forgot to remove it. Apologies. I'll
get it removed when I get back to my desk.

File: components/hab/plan.sh (hunk 1 above)
Comment (eeyun) [reply components/hab/plan.sh]: I think this might be mistakenly added?

Comment (baumanj) [reply components/hab/plan.sh]: Verily 
https://github.com/habitat-sh/habitat/pull/5491#issuecomment-414700519

**Pull request merged**